pkgs/servers/hydron/default.nix

```
@@ -1,22 +1,24 @@
 { lib, stdenv, buildGoPackage, fetchFromGitHub, pkgconfig, ffmpeg-full, graphicsmagick
 , quicktemplate, go-bindata, easyjson }:
+{ lib, stdenv, buildGoModule, fetchFromGitHub, pkg-config, ffmpeg, graphicsmagick }:
```

```suggestion
{ lib, stdenv, buildGoModule, fetchFromGitHub, pkg-config, ffmpeg }:
```

graphicsmagick is no longer used.